pkgs/top-level/all-packages.nix

```
@@ -17119,7 +17119,7 @@ in
 
   nettools_mptcp = callPackage ../os-specific/linux/net-tools/mptcp.nix { };
 
-  nftables = callPackage ../os-specific/linux/nftables { };
+  nftables = callPackage ../os-specific/linux/nftables { withXtables = true; };
```

Can you move the new default to the file instead?
all-packages.nix is hard to navigate:

```suggestion
  nftables = callPackage ../os-specific/linux/nftables { };
```

---

Done!